src/test/java/ttt/BoardTest.java

```diff
@@ -2,6 +2,10 @@
 
 import org.junit.Test;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
```

Comment:
```
import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;
```

These classes are not used and can be removed.

_If there are unused imports you can always optimise them using a shortcut `control + option + o`_